testbeam_analysis/examples/kalman_filter.py

```diff
@@ -104,25 +104,33 @@ def straight_line(x, slope, offset):
     if not os.path.exists(output_folder):
         os.makedirs(output_folder)
 
-    # measurements: (x, y, z) hit positions, taken from event 2518600 of run 118
-    measurements = np.array([[[-1229.22372954, 2828.19616302, 0.],
-                              [-1254.51224282, 2827.4291421, 29900.],
-                              [-1285.6117892, 2822.34536687, 60300.],
-                              [-1311.31083616, 2823.56121414, 82100.],
-                              [-1335.8529645, 2828.43359043, 118700.],
-                              [-1357.81872222, 2840.86947964, 160700.],
-                              [-1396.35698339, 2843.76799577, 197800.]]])
-
-    # copy of measurements for plotting, needed if actual measurements contains NaNs
-    measurements_2 = np.array([[[-1229.22372954, 2828.19616302, 0.],
-                              [-1254.51224282, 2827.4291421, 29900.],
-                              [-1285.6117892, 2822.34536687, 60300.],
-                              [-1311.31083616, 2823.56121414, 82100.],
-                              [-1335.8529645, 2828.43359043, 118700.],
-                              [-1357.81872222, 2840.86947964, 160700.],
-                              [-1396.35698339, 2843.76799577, 197800.]]])
     # pixel size of sensor
     pixel_size = np.array([(18.5, 18.5), (18.5, 18.5), (18.5, 18.5), (18.5, 18.5), (18.5, 18.5), (18.5, 18.5), (250., 50.)])
+    # number of pixels of each sensor
+    n_pixels = np.array([(576, 1152), (576, 1152), (576, 1152), (576, 1152), (576, 1152), (576, 1152), (80, 336)])
+
+    # pixel resolution: need error on measurements. For real data, the error comes from
+    # the cluster position errors which depends on cluster size
+    pixel_resolution = pixel_size / np.sqrt(12)
+
+    # measurements: (x, y, z, xerr, yerr), taken from event 2518600 of run 118
```

Comment:
Where it is exactly comming from is not so important. Just say data from measurement.